test/test-lint.ts

```diff
@@ -25,103 +25,108 @@ import { nop } from '../src/util';
 
 import { withFixtures } from './fixtures';
 
-const OPTIONS: Options = {
-  gtsRootDir: path.resolve(__dirname, '../..'),
-  targetRootDir: './',
-  dryRun: false,
-  yes: false,
-  no: false,
-  logger: { log: nop, error: nop, dir: nop },
-};
+describe('lint', () => {
+  const OPTIONS: Options = {
+    gtsRootDir: path.resolve(__dirname, '../..'),
+    targetRootDir: './',
+    dryRun: false,
+    yes: false,
+    no: false,
+    logger: { log: nop, error: nop, dir: nop },
+  };
 
-const BAD_CODE = `throw 'hello world';`;
-const GOOD_CODE = `throw new Error('hello world');`;
+  const BAD_CODE = `throw 'hello world';`;
+  const GOOD_CODE = `throw new Error('hello world');`;
 
-// missing semicolon, array-type simple.
-const FIXABLE_CODE = 'const x : Array<string> = []';
-const FIXABLE_CODE_FIXED = 'const x : string[] = [];';
+  // missing semicolon, array-type simple.
+  const FIXABLE_CODE = 'const x : Array<string> = []';
+  const FIXABLE_CODE_FIXED = 'const x : string[] = [];';
 
-test.serial('createProgram should return an object', t => {
-  return withFixtures({ 'tsconfig.json': '{}' }, async () => {
-    const program = lint.createProgram(OPTIONS);
-    t.truthy(program);
+  it('createProgram should return an object', () => {
+    return withFixtures({ 'tsconfig.json': '{}' }, async () => {
+      const program = lint.createProgram(OPTIONS);
+      assert.ok(program);
+    });
   });
-});
 
-test.serial('lint should return true on good code', t => {
-  return withFixtures(
-    {
-      'tsconfig.json': JSON.stringify({ files: ['a.ts'] }),
-      'a.ts': GOOD_CODE,
-    },
-    async () => {
-      const okay = lint.lint(OPTIONS);
-      t.is(okay, true);
-    }
-  );
-});
+  it('lint should return true on good code', () => {
+    return withFixtures(
+      {
+        'tsconfig.json': JSON.stringify({ files: ['a.ts'] }),
+        'a.ts': GOOD_CODE,
+      },
+      async () => {
+        const okay = lint.lint(OPTIONS);
+        assert.strictEqual(okay, true);
```

Comment:
Here and elsewhere, use `assert.ok`?

Reply:
I went the other way.  I prefer to use `assert.strictEqual` to verify the object returned is actually a `bool`, and not a random object or something.  I made this consistent throughout.

Reply:
We already get guarantees on return types from TypeScript, IMO we should not duplicate that check in our tests. But this is fine, the outcome is the same.